src/core/moves/MetropolisHastingsMove.cpp

```diff
@@ -258,58 +258,73 @@ void MetropolisHastingsMove::performHillClimbingMove( double lHeat, double pHeat
 
 }
 
+std::map<const DagNode*, double> getNodePrs(const std::vector<DagNode*>& nodes, const RbOrderedSet<DagNode*>& affected_nodes)
+{
+    std::map<const DagNode*, double> Prs;
+    for(auto node: views::concat(nodes, affected_nodes))
+	Prs.insert({node, node->getLnProbability()});
+    return Prs;
+}
 
+constexpr double rel_err_threshhold = 1.0e-11;
```

Comment:
I wonder if these two should not be into settings as well ?
or at least somewhere that is a bit more discoverable than in the middle of a random file ?